website/src/components/MacroComponents/MacrobenchMobile/MacrobenchMobile.jsx

```diff
@@ -14,108 +14,213 @@ See the License for the specific language governing permissions and
+            }}
+          >
+            here
+          </Link>{" "}
+          to see the query plans
         </div>
-    );
+      </div>
+      <div className="macrobenchMobile__container flex">
+        <div className="macrobench__Sidebar flex--column">
+          <span>QPS Total</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>QPS Reads</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>QPS Writes</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>QPS Other</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>TPS</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Latency</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Errors</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Reconnects</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Time</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Threads</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Total CPU time</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>CPU time vtgate</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>CPU time vttablet</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Total Allocs bytes</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Allocs bytes vtgate</span>
+          <figure className="macrobench__Sidebar__line"></figure>
+          <span>Allocs bytes vttablet</span>
+        </div>
+
+        <div className="macrobench__component__container flex">
+          <Swiper
+            effect={"cards"}
+            grabCursor={true}
+            modules={[EffectCards]}
+            onSlideChange={handleSlideChange}
+            initialSlide={currentSlideIndexMobile}
+            className="mySwiper"
+          >
+            <SwiperSlide>
+              <div className="macrobench__data flex--column">
+                <h4>Impoved by %</h4>
+                <span>{fixed(data.diff.Diff.qps.total, 0)}</span>
+                <span>{fixed(data.diff.Diff.qps.reads, 0)}</span>
+                <span>{fixed(data.diff.Diff.qps.writes, 0)}</span>
+                <span>{fixed(data.diff.Diff.qps.other, 0)}</span>
+                <span>{fixed(data.diff.Diff.tps, 0)}</span>
+                <span>{fixed(data.diff.Diff.latency, 0)}</span>
+                <span>{fixed(data.diff.Diff.errors, 0)}</span>
+                <span>{fixed(data.diff.Diff.reconnects, 0)}</span>
+                <span>{fixed(data.diff.Diff.time, 0)}</span>
+                <span>{fixed(data.diff.Diff.threads, 0)}</span>
```

Comment:
For all float values in this page (not only in improved by) we should used `fixed` with `2`, same as in the desktop version.